### DISCUSSION TITLE
Is there a way to use text object to move by words within a camel case word

### DISCUSSION THREAD

**Discussion by NamekMaster** [JetBrains/ideavim#915] [Q&A]
I found a guide about how to use text object in [here](https://blog.carbonfive.com/vim-text-objects-the-definitive-guide/), It said 

> [CamelCaseMotion](https://github.com/bkad/CamelCaseMotion) provides a text object to move by words within a camel or snake-cased word.

`ci,w` is used to delete the word within camel case word

But it seems ideavim doesn't support that plugin, is there any way to do that? Looks like it is useful. I found that ideavim supports `[w` motion to move by words within camel case word, but when I type `di[w`, it deletes the word inside `[]`

**Comment by AlexPl292** [comment]
Hi, we don't support the camelCase plugin yet, but you can find some ideas here: https://youtrack.jetbrains.com/issue/VIM-1984/Support-CamelCaseMotion.vim-plugin
